packages/cozy-harvest-lib/src/components/cards/AppLinkCard.jsx

```diff
@@ -44,7 +45,8 @@ const AppLinkCard = ({
           href={generateWebLink({
             cozyUrl: cozyURL.origin,
             slug,
-            nativePath: path
+            nativePath: path,
+            subDomainType
           })}
         >
           {({ onClick, href }) => (
```

Comment:
Should we add a test for this component ?

Reply:
i don't really see what could be tested here except a snapshot of the component. And since it's a very straightforward component with no logic I don't think it's useful, I thought the important test to add was checking that the links appear as we want.